### PR TITLE
Issue #516 - Composite Tracks need to be added to Debrief Schema

### PR DESCRIPTION
**Pull request by pecko**

### CODE REVIEW

File: org.mwc.debrief.core/schema/debrief_plot.xsd [hunk 1]
```diff
@@ -286,6 +287,51 @@
...
+		<xs:sequence>
+			<xs:element minOccurs="0" maxOccurs="unbounded" name="fix" type="fix" />
+			<xs:element minOccurs="0" name="WorldDistance" type="unitsValueType" />
+			<xs:element minOccurs="0" name="Speed" type="unitsValueType" />
+			<xs:element minOccurs="0" name="colour" type="colourType" />
+			<xs:element minOccurs="0" name="Duration" type="unitsValueType" />
+		</xs:sequence>
+		<xs:attribute name="LineStyle" type="xs:string" />
+		<xs:attribute name="Name" type="xs:string" />
+		<xs:attribute name="calcModel" type="xs:string" />
```
Comment (IanMayo) [inline org.mwc.debrief.core/schema/debrief_plot.xsd]: there is actually a limited set of model types, as recorded here:
https://github.com/debrief/debrief/blob/master/org.mwc.cmap.legacy/src/MWC/GUI/Properties/PlanningLegCalcModelPropertyEditor.java#L90

Comment (pecko) [inline org.mwc.debrief.core/schema/debrief_plot.xsd]: Will fix it today later.

**Comment by IanMayo** [comment]
@pecko  - have made minor change to how Composite Tracks are exported, so please remember to *Pull* before you push your changes,
thx.

**Comment by pecko** [comment]
I have done that.

Notice: "export the Origin before the legs, that's the logical order" can affect old plots.

**Comment by IanMayo** [comment]
"Can affect old plots"

The only effect I can think of is that if we save an old file, the order of the elements will change.  But, there won't be any software breakages, will there?

**Comment by pecko** [comment]
No, it won't. The schema validation for old plots will not work. It will not affect current software.

**Comment by IanMayo** [comment]
Duh, yes, of course.

Still, it does seem the right thing to do.

**Comment by pecko** [comment]
Agree. It is minor issue.

**Pull request merged**